packages/oasgraph/README.md

```diff
@@ -149,13 +149,17 @@ The options object can contain the following properties:
+* `requestOptions` (type: `object`, default: `{}`): Additional [options](https://github.com/request/request#requestoptions-callback), provided by the  [`Request` module](https://github.com/request/request), that can be used to configure the HTTP calls that powers the generated GraphQL resolvers. A common use case for this option is to set up a web proxy with the `proxy` field.
+
+* `provideErrorExtensions` (type: `boolean`, default: `true`): If a query cannot fulfilled, GraphQL will provide a [list of error objects](https://graphql.github.io/graphql-spec/draft/#sec-Errors) for all fields that could not be resolved. OASGraph will add an extensions to all error objects resulting from REST calls that did not return successful HTTP codes (i.e. 200-299). These extensions contain data about the REST call (e.g. the method, path, status code, response headers, and response body). This data can be useful for debugging but it can also _unintentionally leak information_. This option prevents the error extensions from being created. 
+
+* `customResolvers` (type: `object`, default: `{}`): OASGraph will normally use REST calls, derived from the OAS, to resolve fields in the generated GraphQL interface. This option allows users to utilize custom resolvers in place of said REST calls. The field that the custom resolver will affect is identifed first by the [title](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#infoObject) of the OAS, then the [path](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#paths-object) of the operation, and lastly the [method](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#path-item-object) of the operation. The `customResolvers` object is triply nested object where the outer key is the title, followed by the path, and finally the method, which points to the [resolver function](https://graphql.org/learn/execution/#root-fields-resolvers) itself. The resolver function can use the parameters `obj`, `args`, `context`, and `info` in order to produce the proper data. Use cases include the resolution of complex relationships between types, implementing performance improvements like caching, or dealing with non-standard authentication requirements. _Note: Because the arguments are provided by the GraphQL interface, they may look different from the [parameters](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#parameterObject) provided by the OAS. For example, they will have [sanitized](https://github.com/Alan-Cha/oasgraph#characteristics) names. The [request body](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#requestBodyObject) will also be contained in the arguments as an [input object type](https://graphql.org/graphql-js/mutations-and-input-types/)._
```

Comment:
OASGraph will normally use REST calls, derived from the OAS, to resolve fields in the generated GraphQL interface. -> OASGraph, per default, creates resolver functions that make REST calls to resolve fields in the generated GraphQL interface.

Reply:
This option allows users to utilize custom resolvers in place of said REST calls. -> This option allows users to provide custom resolver functions to be used in place of said ones created by OASGraph.

Reply:
The `customResolvers` object is triply nested object -> The `customResolvers` object is thus a triply nested object

Reply:
The resolver function can use the parameters `obj`, `args`, `context`, and `info` in order to produce the proper data. -> The resolver function can use the parameters `obj`, `args`, `context`, and `info` in order to produce the proper data, as do [standard resolver functions in GraphQL](https://graphql.org/learn/execution/#root-fields-resolvers).

Reply:
provided by the OAS. -> defined in the OAS.